aca_preview/preview.py

```diff
@@ -336,3 +340,66 @@ def check_bright_guide_for_ers(self, n_bright_req=3, bright_lim=9.0):
         n_bright = np.count_nonzero(self.guides['mag'] < bright_lim)
         if not is_OR and n_bright < n_bright_req:
             self.add_message('critical', f'ER bright stars: only {n_bright} stars brighter than {bright_lim}')
+
+    def check_enough_guide_for_ers(self, n_required=8):
+        """
+        Warn on ERs with fewer than n_required (8) guide stars.
+        """
+        obsid = float(self.obsid)
+        is_OR = abs(obsid) < 38000
+        if not is_OR and len(self.guides) < n_required:
+            self.add_message('critical', f'ER guides stars: only {len(self.guides)} stars')
+
+    def check_guide_pos_errs(self):
+        """
+        Warn on stars with larger POS_ERR (warning at 1" critical at 2")
+        """
+        for star in self.guides:
+            agasc_id = star['id']
+            # POS_ERR is in milliarcsecs in the table
+            pos_err = star['POS_ERR'] * 0.001
+            for limit, category in ((2.0, 'critical'),
+                                    (1.0, 'warning')):
+                if pos_err > limit:
+                    self.add_message(category, f'Guide star {agasc_id} has POS_ERR {pos_err:.2f}, limit {limit} arcsec')
+                    break
+
+    def check_imposters(self):
+        """
+        Warn on stars with larger imposter centroid offsets
+        """
+
+        # Borrow the imposter offset method from starcheck
+        def imposter_offset(cand_mag, imposter_mag):
+                """
+                For a given candidate star and the pseudomagnitude of the brightest 2x2 imposter
+                calculate the max offset of the imposter counts are at the edge of the 6x6
+                (as if they were in one pixel).  This is somewhat the inverse of proseco.get_pixmag_for_offset
+                """
+                cand_counts = mag_to_count_rate(cand_mag)
+                spoil_counts = mag_to_count_rate(imposter_mag)
+                return spoil_counts * 3 * 5 / (spoil_counts + cand_counts)
+
+        for star in self.guides:
+            agasc_id = star['id']
+            imp_offset = imposter_offset(star['mag'], star['imp_mag'])
+            for limit, category in ((4.0, 'critical'),
+                                    (2.5, 'warning')):
+                if imp_offset > limit:
+                    self.add_message(category, f'Guide star {agasc_id} imposter offset {imp_offset:.1f}, limit {limit} arcsec')
+                    break
+
+    def check_too_bright_guide(self):
+        """
+        Warn on guide stars that may be too bright.
```

Comment:
For this and other method docstrings, put a line break between the one-line method description and the details.    In general, in emacs and hit `ESC-q` to format correctly.  In this case it would be better to format this as a list like:
```
- Critical warn if MAG_ACA_ERR used in selection is less than 0.1
- Critical warn if within 3 * mag_err ...
- ..
```